docs/appendices/release-notes/5.10.11.rst

```diff
@@ -47,4 +47,8 @@ See the :ref:`version_5.10.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Fixed an issue with choosing the correct super data type when storing nested
+  array of object values with different data types in the same column into
+  a dynamic object typed column. The issue caused the bulk/batch insert to
+  partially fail, some rows were inserted while others were not, depending on
+  which array of object value was processed first (kinda random).
```

Comment:
```suggestion
- Fixed an issue which caused inserts of object arrays with columns having
  mixed types to partially fail. It used the type of the first value seen, and
  failed for the values with a different type.

  The new behavior is that it will use the type with the higher precendence,
  and cast the other values if possible.
```

Not sure if it is an improvement. Fine to use your version too - tried to focus a bit more on the user facing behavior.

Reply:
I think it is, like it, thx!